charts/exporter/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ssl_exporter
+description: A Helm chart for SSL Exporter
+type: application
+version: 0.1.0
+appVersion: 2.3.1
```

Comment:
These will need to be bumped as new versions are released. Ideally this would be automated I guess, but just as a start could you add it to the [README](https://github.com/ribbybibby/ssl_exporter#release-process) so there's at least a reminder for me?

Reply:
> Ideally this would be automated I guess

Unfortunately, I'm not familiar with Github Actions, yet.
I usually use `sed` on `Chart.yaml` to update app/chart version.

> but just as a start could you add it to the README so there's at least a reminder for me?

ok, np